projects/ng-core-tester/src/app/record/editor/schema.json

```diff
@@ -1034,10 +1034,6 @@
       "type": "array",
       "minItems": 1,
       "uniqueItems": true,
-      "default": [
```

Comment:
Why?

Reply:
I removed the “default” to make sure I had a “Select...” when the field was displayed.